tests/legacy/test_read_exceptions.py

```diff
@@ -137,9 +137,6 @@ def test_timeout_raw(self, generate_task, real_x_series_device):
         number_of_samples_expected = clocks_to_give - samples_to_read
         assert timeout_exception.value.samps_per_chan_read == number_of_samples_expected
 
-    @pytest.mark.grpc_xfail(
-        reason="AB#2420742: gRPC interpreter doesn't retain the initial array values when converting flattened array into numpy array."
-    )
     def test_timeout_stream(self, generate_task, real_x_series_device):
```

Comment:
FYI, I can't think of a strong reason to keep a single-channel version or parameterize on # channels